reddit.js

```diff
@@ -1,5 +1,38 @@
 var reddit = require("./data.json")
+// console.log(reddit["data"]);
 
 // Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+// forEach - cosole.log all titles
+reddit.data.children.forEach(function (titles) {
```

Comment:
Here, perhaps ```child``` would be a better parameter name than ```titles```, as forEach is iterating over each *child* object in the ```data.children``` array.